### DISCUSSION TITLE
How can I manipulate the position update interval?

### DISCUSSION THREAD

**Discussion by UkeFan** [osmandapp/OsmAnd#20239] [Q&A]
Hi! On my old Galaxy S7 the new osmand+ hangs quite often for many seconds on one point folowing an track. After maybe 100m it quickly updates to the current position. 
I can't find anymore a/the setting for that interval!?

**Comment by mikehgentry** [comment]
Try disabling 'animate own position'

#16081

**Reply by UkeFan** [reply]
Thanks, "unfortunately" it was always de-actived. So this can't help

**Reply by dmpr0** [reply]
How it works if **Animate own position** is turned on?

**Reply by ThomasMeister123** [reply]
I have the same problems for years...

**Reply by ThomasMeister123** [reply]
https://github.com/osmandapp/OsmAnd/discussions/18706

**Reply by ThomasMeister123** [reply]
Here an example of this behavior.

Real position ist at (1). OSMand already knows this (see green track-recording-line). Also the rotation of the map is for the real position.

And thats not a "worse" example. Sometimes the "gap" is much longer

![2024-11-12 12_08_23-XnView -  A2 png ___ 720x1280 ___ Verschluss_  ___ Blende_  ___ Iso_ ](https://github.com/user-attachments/assets/ee47dad0-3185-43ae-9da7-3eefff08130f)
